qiskit/assembler/assemble_schedules.py

```diff
@@ -76,7 +76,9 @@ def assemble_schedules(schedules, qobj_id, qobj_header, run_config):
                     # verify all acquires satisfy meas_map
                     _validate_meas_map(instruction, meas_map)
 
-            qobj_instructions.append(instruction_converter(shift, instruction))
+            converted_instruction = instruction_converter(shift, instruction)
+            if converted_instruction:
```

Comment:
There's some implicit logic here: assume `converted_instruction` will only evaluate to `False` if `instruction` is a `Delay`. Can we check the type of `instruction`, and only convert for non delays? Or in some other way make our failure routes or expected behavior more explicit?